src/common/exception.cpp

```diff
@@ -27,5 +27,18 @@ std::string ExceptionMessage::overLargeStringValueException(const std::string& l
         "Maximum length of strings is 4096. Input string's length is {}.", length);
 }
 
+std::string ExceptionMessage::violateUniquenessOfRelAdjColumn(const std::string& tableName,
```

Comment:
inline these two exceptions messages since they are only one line long.

Reply:
They are kept in cpp file to avoid include `StringUtils` in header.